### ISSUE TITLE
CREATE TYPE syntax error

### ISSUE THREAD

**Issue by siggemannen** [MicrosoftDocs/sql-docs#9463]
Hello. On this page: https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/statements/create-type-transact-sql.md
the syntax to create type is specified as follows:
```sql
CREATE TYPE [ schema_name. ] type_name  
{   
    [
      FROM base_type   
      [ ( precision [ , scale ] ) ]  
      [ NULL | NOT NULL ]
    ]
    | EXTERNAL NAME assembly_name [ .class_name ]   
    | AS TABLE ( { <column_definition> | <computed_column_definition> [ ,... n ] }
      [ <table_constraint> ] [ ,... n ]    
      [ <table_index> ] [ ,... n ] } )
 
} [ ; ]  
```
But from base_type isn't completely optional so it should probably be:

```sql
CREATE TYPE [ schema_name. ] type_name  
{   
      FROM base_type   
      [ ( precision [ , scale ] ) ]  
      [ NULL | NOT NULL ]
    | EXTERNAL NAME assembly_name [ .class_name ]   
    | AS TABLE ( { <column_definition> | <computed_column_definition> [ ,... n ] }
      [ <table_constraint> ] [ ,... n ]    
      [ <table_index> ] [ ,... n ] } )
 
} [ ; ]  
```

**Comment by rwestMSFT** [comment]
@siggemannen Do you mind creating a pull request with this change?

**Comment by rwestMSFT** [comment]
@WilliamAntonRohm This is not my article.

**Comment by siggemannen** [comment]
@rwestMSFT i can try to fix one up!

**Comment by rwestMSFT** [comment]
Resolved by #9466.
